hypertrace-graphql-labels-schema-impl/src/main/java/org/hypertrace/graphql/label/dao/LabelResponseConverter.java

```diff
@@ -60,5 +66,36 @@ private Label convertLabel(org.hypertrace.label.config.service.v1.Label label) {
     String key;
     String color;
     String description;
+    Map<String, List<LabeledEntity>> labeledEntitiesMap;
```

Comment:
would make sense to hold a `Map<String, LabeledEntityResultSet>` I think

Reply:
Sure